develop/smart-contracts/connect-to-polkadot.md

```diff
@@ -86,9 +56,9 @@ Developers can leverage Asset Hub across diverse networks, from TestNets to Main
 
 To get started with MetaMask, you need to install the [MetaMask extension](https://metamask.io/download/){target=\_blank} and add it to the browser. Once you install MetaMask, you can set up a new wallet and securely store your seed phrase. This phrase is crucial for recovery in case you lose access.
 
-For example, to connect to the Westend Asset Hub TestNet via MetaMask, you need to follow these steps:
+For example, to connect to the Westend Hub TestNet via MetaMask, you need to follow these steps:
```

Comment:
Sorry I know I originally suggested this change, but it seems like in this instance we should be saying connect to Westend Asset Hub, especially since the network configs below say `Network name - Asset-Hub Westend TestNet`